### ISSUE TITLE
package manager file?

### ISSUE THREAD

**Issue by sje30** [dvdvgt/typst-letter#1]
Hi,

Thank you for this letter template.  

Have you seen that typst now has a package manager?  Details are at:

https://typst.app/blog/2023/package-manager/

and as far as I can understand would be as simple as adding some metadata in typst.toml looking like

```
[package]
name = "typst-letter"
version = "0.1.0"
entrypoint = "template.typ"
```

and then making a PR to https://github.com/typst/packages

**Comment by sje30** [comment]
p.s. the guidelines suggest using a specific name for the package, e.g. 'letter-din5008' rather than 'typst-letter'.

**Comment by sje30** [comment]
as I didn't want to move this into the package structure without your permission, for now I've added it as a local package on my computer.  To this, I still needed a typst.toml file, so here it is if you, or others, need it.

```
[package]
name = "din5008"
version = "0.1.0"
entrypoint = "din5008.typ"
authors = ["David Voigt"]
license = "MIT"
description = "Letter template conforming to DIN5008."
```

and you can see I've gone with din5008 as a more specific package name.

This can then be included from a typst file using:
```
#import "@local/din5008:0.1.0": *
```

**Comment by dvdvgt** [comment]
Hi @sje30

I am happy to hear that you apparently found use for this template. Thanks for already supplying the needed `.toml` file. I will happily add it to this repository and try to submit it to the official package “registry”.

**Comment by sje30** [comment]
Yes, I've used it for several letters already, and I find it very convenient (and was looking for an alternative to cumbersome latex letters.)

**Comment by dvdvgt** [comment]
> Yes, I've used it for several letters already, and I find it very convenient (and was looking for an alternative to cumbersome latex letters.)

Glad to hear that! Is there anything you encountered, you would like to see changed before I submit it as a package? Some annoyances or further configuration options?

**Comment by sje30** [comment]
Perhaps the only issue I hit so far is that the "Name Surname" of the writer is the same at the top of the page and when signing off.  I had one letter where I wished I could write something other than my name at the bottom of the page (I wanted to write "Me, on behalf of x,y,z").

A logo might be useful, but then you have to think about its size, location, etc, which will be hard to please everyone with.  I'd suggest keeping it simple for now and see what people ask for.

**Comment by CGMossa** [comment]
Hey! Can I ask about the status of getting this to be a package?
I have looked and it have not been submitted yet.

**Comment by CGMossa** [comment]
Oh right! It is recommended not to use packages for templates, for now! That's the reason.

Sorry 🙏 but at least the issue is not somewhat updated.
